pylint/lint/pylinter.py

```diff
@@ -709,7 +710,15 @@ def _get_asts(
         """Get the AST for all given FileItems."""
         ast_per_fileitem: dict[FileItem, nodes.Module | None] = {}
 
+        # For progress reports, it would be nice to say "item 1 of N",
+        # but we can't get the total count of filenames without
+        # materializing the list, or creating a second iterator and
+        # iterating.
+        progress_reporter = get_progress_reporter(self.verbose)
+        progress_reporter.print_message("Get ASTs.")
```

Comment:
Same as above, this should probably be its own method instead of passing the message itself

Reply:
Right, keeps this code clearer.

Reply:
Pushed, now is `progress_reporter.start_get_asts()`

Reply:
Done per later feedback.